docs/pages/docs/guides/choosing-a-database.mdx

```diff
@@ -0,0 +1,52 @@
+import { Markdown } from '../../../components/Markdown';
+import { Well } from '../../../components/primitives/Well';
+import { RelatedContent } from '../../../components/RelatedContent';
+
+# Choosing the right Database
+
+Keystone supports Postgres, MySQL and SQLite as database engines for your backend. There might be different reasons why you would choose one over the other.
+
+This page seeks to clarify some of the differences in the databases supported by Keystone.
+
+!> **Note:** SQLite is not recommended in production unless for readonly senarios like the [embedded Keystone](../walkthroughs/embedded-mode-with-sqlite-nextjs) example
+
+## Case Sensitivity
+
+By default, Postgres is case sensitive, while MySQL and SQLite are not case sensitive. This means when filtering and sorting the results will differ depending on the database type and collation used.
+When using `StringFilter`in a GraphQL query the `mode` defaults to that of the database - case sensitive for Postgres, and collation dependent for MySQL, you can make a query case insensitive by using `mode: insensitive`, there is no way to make a `StringQuery` case sensitive if your database is not case sensitive.
+
+See the [Case Sensitivity Prisma Docs](https://www.prisma.io/docs/concepts/components/prisma-client/case-sensitivity) for more information.
+
+## Default Types
+
+Prisma has different default types for each database used, for example, the Keystone `text` field type uses the Prisma type of `String`,
+for Postgres, Prisma uses the `text` column type, and for MySQL, it uses the `varchar(191)` column type. For more details on the `field` type differences
+see [Primsa Schema Reference](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#model-field-scalar-types).
+
+The Keystone `text` field supports the `db.nativeType` option, allowing you to override this - [Fields API](../apis/fields). Over time this option will be added to other field types.
+
+## Prisma Resources
+
+- [Database Features](https://www.prisma.io/docs/reference/database-reference/database-features)
+- [Supported Databases](https://www.prisma.io/docs/reference/database-reference/supported-databases)
+
+!> **Note**: Keystone currently supports Postgres, MySQL, SQLite
```

Comment:
Not sure we need this due to the presence in the top of page callout.

If you'd like to keep it, suggest including `Prisma` somewhere to situate it in the Prisma H2 in a more obvious way.

Reply:
Thanks @raveling I have just rewritten this section